recipe/build.sh

```diff
@@ -4,6 +4,7 @@ set -e
 . ${RECIPE_DIR}/pg.sh
 
 export CPPBIN="${CPP}"
+export PG_SYSROOT=$CONDA_BUILD_SYSROOT
```

Comment:
@isuruf, do we also want this to be:
```suggestion
export PG_SYSROOT="undefined"
```
like in https://github.com/conda-forge/postgresql-feedstock/pull/89?